AutoMapper.OData.EFCore.Tests/GetQueryTests.cs

```diff
@@ -149,7 +149,7 @@ public async void OpsTenantExpandBuildingsNestedFilterEqUsingThisParameterWithNo
 
             static void Test(ICollection<OpsTenant> collection)
             {
-                Assert.Equal(2, collection.Count);
+                Assert.Equal(3, collection.Count);
```

Comment:
Leaving existing tests untouched gives us more confidence that the previous logic still works.  A standalone in-memory queryable like [this one](https://github.com/AutoMapper/AutoMapper.Extensions.OData/blob/a19b58a1ab5980223eef7aa3b50f25ea974296ab/AutoMapper.OData.EFCore.Tests/GetQueryTests.cs#L851) should work for your case - allows for testing both root and child collections.